package.json

```diff
@@ -90,6 +90,10 @@
       {
         "name": "2021-09-release",
         "prerelease": true
+      },
```

Comment:
Should this not be removed?

Reply:
Why?

Reply:
Truee, but kinda "wasteful" to create a new  PR just for that now that you are already changing it 😆